docs/docs/stylesheets/extra.css

```diff
@@ -22,3 +24,18 @@
 .md-typeset .md-button:hover, .md-button--primary:hover {
     border-color: white;
 }
+
+.md-typeset .admonition.wip,
+.md-typeset details.wip {
+border-color: rgb(160, 45, 0);
+}
+.md-typeset .wip > .admonition-title,
+.md-typeset .wip > summary {
+background-color: rgb(255, 195, 172);
+}
+.md-typeset .wip > .admonition-title::before,
+.md-typeset .wip > summary::before {
+background-color: rgb(255, 77, 6);
```

Comment:
we could add [Malva](https://github.com/g-plane/malva) to our dprint config for formatting the CSS, wdyt? doesn't have to be part of this PR of course

Reply:
I support it